modules/ItemLevel.lua

```diff
@@ -324,6 +326,13 @@ do
 		[100] = { 136, 143 }, -- Warlords of Draenor
 		[110] = { 164, 250 }, -- Legion
 		[120] = { 400, 485 }, -- Battle for Azeroth
+]]--
+		[10] = { 1, 20 }, -- Leveling
+		[20] = { 25, 40 }, -- Leveling
+		[30] = { 45, 60 }, -- Leveling
+		[40] = { 65, 80 }, -- Leveling
+		[50] = { 85, 168 }, -- Leveling
+		[60] = { 170, 300 }, -- Battle for Azeroth
```

Comment:
Are these ranges exposed by the API somewhere? Or player testing?

Reply:
No, it's handmade testing :) 
Since the armory/wowhead/wowdb are note accurately updated with new itemlevel, i made this by testing!
I'm currently leveling an alt to get more logical ranges (taking the range from the heirloom gear and dungeon loots at each max level to test)! But since EU have authentication server problem i have trouble connecting when i come back from work, i took the gear of my dugeon different alts to make logical ranges :) ....
I have to test if range can be overlapping though!

I have search for quite some time but did not found any relation between level and itemlevel....
https://www.wowhead.com/guides/shadowlands-item-level-squish
i found that...
But for example it does not take for account the legendary cloack which is 500, and the neckalce that can go up to 530+
And since you can choose whichever expansion you want to level in, i think this graph is helping but not totally accurate.